### PR TITLE
Chore/screen orientation

### PR DESCRIPTION
**Pull request by rudibravo**
@Rise-Vision/delivery please review

### CODE REVIEW

**Comment by rudibravo** [comment]
[(3) ChrOS: screen orientation](https://trello.com/c/Z96ff6QN/4214-3-chros-screen-orientation)

File: test/unit/orientation.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+describe('Orientation', () => {
+
+  afterEach(() => chrome.flush());
+
+  afterEach(() => sandbox.restore());
+
+  beforeEach(() => sandbox.stub(logger, 'log'));
+
+  it('should rotate', () => {
+    chrome.storage.local.get.yields({});
```
Comment (andrecardoso) [inline test/unit/orientation.js]: Why do you need to mock local storage?

Comment (rudibravo) [reply test/unit/orientation.js]: @andrecardoso good catch, I don't.

**Comment by rudibravo** [comment]
@andrecardoso removed the unnecessary lines.

**Comment by andrecardoso** [comment]
@rudibravo looks good. Thanks

**Pull request merged**